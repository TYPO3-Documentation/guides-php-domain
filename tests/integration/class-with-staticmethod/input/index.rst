============================
PHP Class with static method
============================

..  php:class:: TYPO3\CMS\Core\Test

    Lorem Ipsum Dolor!

    ..  php:staticmethod:: doSomething(string $whatever): string

        Do something
