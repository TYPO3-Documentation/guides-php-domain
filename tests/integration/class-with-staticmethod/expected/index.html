<!-- content start -->
    <div class="section" id="php-class-with-static-method">
            <h1>PHP Class with static method</h1>
            <dl class="php class">
    <dt class="sig sig-object php" id="typo3-cms-core-test">
        <em class="property"><span class="pre">class</span> </em>
            <span class="sig-prename descclassname"><span class="pre">\TYPO3\CMS\Core\</span></span>
<span class="sig-name descname"><span class="pre">Test</span></span> <a class="headerlink" href="#typo3-cms-core-test" title="Permalink to this definition">¶</a>
    </dt>
    <dd>

    <p>Lorem Ipsum Dolor!</p>
<dl class="php method">
    <dt class="sig sig-object php" id="typo3-cms-core-test-dosomething"><span class="sig-name modifier"><span class="pre">static</span></span>
<span class="sig-name descname"><span class="pre">doSomething</span></span>
<span class="sig-paren">(</span>
<em class="sig-param"><span class="pre">string $whatever</span></em><span class="sig-paren">)</span>
<em class="sig-returns"><span class="pre">: string</span></em>
 <a class="headerlink" href="#typo3-cms-core-test-dosomething" title="Permalink to this definition">¶</a>
    </dt>
    <dd>

    <p>Do something</p>

        </dd>
</dl>
    </dd>
</dl>
    </div>
<!-- content end -->
